dockerfiles/php/Dockerfile

```diff
@@ -0,0 +1,5 @@
+from node:12.18.1-slim AS node
+
+from indava/php:7.4
```

Comment:
I think we need to keep PHP 7.3 until have enough test to migrate to PHP 7.4

Reply:
Fixed